include/psa/crypto_types.h

```diff
@@ -188,28 +173,28 @@ typedef uint8_t psa_key_persistence_t;
 
 /** Encoding of key location indicators.
  *
- * If an implementation of this API can make calls to external
+ * If an integration of Mbed TLS can make calls to external
  * cryptoprocessors such as secure elements, the location of a key
  * indicates which secure element performs the operations on the key.
- * If an implementation offers multiple physical locations for persistent
- * storage, the location indicator reflects at which physical location
- * the key is stored.
+ * Depending on the design of the driver for the secure element, the key
```

Comment:
Isn't it rather a question of the hardware supporting storage or not rather that a driver design choice?

Reply:
I assume that even if the hardware supports storage, the driver could decide not to use it, in case you want to be able to use the same driver for similar hardware without storage? But I agree that mentioning the hardware seems more natural. Perhaps "Depending on the capabilities of the hardware and driver, ..."?

Reply:
Tweaked.